modules/applications/pages/ui-builder/how-to-configure-https.adoc

```diff
@@ -41,20 +41,22 @@ services:
     image: ...
     environment: ...
     volumes:
-      - type: bind
-        source: <path_on_docker_host>/nginx.conf
-        target: /etc/nginx/templates/nginx.conf.template
-      - type: bind
-        source: <path_on_docker_host>/https.conf
-        target: /etc/nginx/snippets/https.conf
+      - <path_on_docker_host>/nginx.conf:/etc/nginx/templates/nginx.conf.template
+      - <path_on_docker_host>/https.conf:/etc/nginx/snippets/https.conf
```

Comment:
**question:** 
Is it normal that is not exactly the same content than our docker-compose.yml in UIB repository ?

ℹ️ In our repo:
```
      - ../nginx/https.conf:/etc/nginx/snippets/https.conf
      - ../nginx/https-redirect.conf:/etc/nginx/snippets/https-redirect.conf
      - ../nginx/cookies.conf:/etc/nginx/snippets/cookies.conf
```